java/code/src/com/redhat/rhn/taskomatic/task/ReportDbUpdateTask.java

```diff
@@ -29,42 +31,14 @@
 import org.quartz.JobExecutionContext;
 import org.quartz.JobExecutionException;
 
-import java.util.List;
 import java.util.Map;
-import java.util.Set;
-import java.util.stream.Collectors;
-import java.util.stream.Stream;
+
 
 
 public class ReportDbUpdateTask extends RhnJavaJob {
 
     private static final int BATCH_SIZE = 100;
 
-    @SuppressWarnings("unchecked")
```

Comment:
Here we have also a BATCH_SIZE where we can use the value from the config.
It is 2 lines above this, but i cannot add the comment to the correct line